guidelines/index.html

```diff
@@ -1,352 +1,320 @@
-    <dt>WCAG 2.0 の編集者 (2008年12月まで):</dt><dd class="p-author h-card vcard" data-editor-id="33602"><span class="p-name fn">Ben Caldwell</span> (Trace R&amp;D Center, University of Wisconsin-Madison)</dd><dd class="p-author h-card vcard" data-editor-id="35436"><span class="p-name fn">Loretta Guarino Reid</span> (Google, Inc.)</dd><dd class="p-author h-card vcard" data-editor-id="3442"><span class="p-name fn">Gregg Vanderheiden</span> (Trace R&amp;D Center, University of Wisconsin-Madison)</dd><dd class="p-author h-card vcard" data-editor-id="4099"><span class="p-name fn">Wendy Chisholm</span> (<abbr title="World Wide Web Consortium">W3C</abbr>)</dd><dd class="p-author h-card vcard" data-editor-id="35537"><span class="p-name fn">John Slatin</span> (Accessibility Institute, University of Texas at Austin)</dd><dd class="p-author h-card vcard" data-editor-id="74028"><span class="p-name fn">Jason White</span> (University of Melbourne)</dd>
-    
-    
-  </dl>
-  <p>
-      
-      公表後に報告された誤りや問題については <a href="https://www.w3.org/WAI/WCAG21/errata/"><strong>errata</strong></a> を確認すること。
-    
-    </p>
-  <p>
-  	<a href="https://www.w3.org/WAI/standards-guidelines/wcag/translations/"> <strong>翻訳版</strong></a>も参照できる。
-    </p>
-    	
-    	<p>この文書は、規定ではないフォーマットでも提供されており、<a href="https://www.w3.org/WAI/WCAG21/versions/">Alternate Versions of Web Content Accessibility Guidelines 2.1</a> より入手できる。</p>
-  
-  <p class="copyright">
-        <a href="https://www.w3.org/Consortium/Legal/ipr-notice#Copyright">Copyright</a> ©
-        2017-2018
+<div class="head">
+    <p class="logos"><a class="logo" href="https://www.w3.org/"><img crossorigin="" alt="W3C" height="48" src="https://www.w3.org/StyleSheets/TR/2021/logos/W3C" width="72">
+  </a></p>
+    <h1 id="title" class="title">Web Content Accessibility Guidelines (WCAG) 2.1</h1> 
+    <p id="w3c-state"><a href="https://www.w3.org/standards/types#REC">W3C Recommendation</a> <time class="dt-published" datetime="2023-09-21">21 September 2023</time></p>
+    <details open="">
+      <summary>この文書の更なる詳細</summary>
+      <dl>
+        <dt>このバージョン:</dt><dd>
+                <a class="u-url" href="https://www.w3.org/TR/2023/REC-WCAG21-20230921/">https://www.w3.org/TR/2023/REC-WCAG21-20230921/</a>
+              </dd>
+        <dt>最新バージョン:</dt><dd>
+                <a href="https://www.w3.org/TR/WCAG21/">https://www.w3.org/TR/WCAG21/</a>
+              </dd>
+        <dt>最新の編集者バージョン:</dt><dd><a href="https://w3c.github.io/wcag/guidelines/22/">https://w3c.github.io/wcag/guidelines/22/</a></dd>
+        <dt>履歴:</dt><dd>
+                    <a href="https://www.w3.org/standards/history/WCAG21/">https://www.w3.org/standards/history/WCAG21/</a>
+                  </dd><dd>
+                    <a href="https://github.com/w3c/wcag/commits/">コミット履歴</a>
```

Comment:
基本的にリソースのリンクは英語のママにしているはず…なので英語に戻します。
```suggestion
                    <a href="https://github.com/w3c/wcag/commits/">Commit history</a>
```